app/renderer/store.js

```diff
@@ -11,52 +10,32 @@ const { createReduxHistory, routerMiddleware, routerReducer } = createReduxHisto
 
 const rootReducer = createRootReducer(routerReducer);
 
-const configureStore = (initialState) => {
-  const middleware = [];
-  let composeEnhancers = compose;
-
-  // Thunk Middleware
-  middleware.push(thunk);
-
-  // Additional development tools
-  if (process.env.NODE_ENV === 'development') {
-    // Logging Middleware
-    const { createLogger } = require('redux-logger');
-    const logger = createLogger({
-      level: 'info',
-      collapsed: true
+export const store = configureStore({
+  reducer: rootReducer,
+  middleware: (getDefaultMiddleware) => {
+    const middleware = getDefaultMiddleware({
+      serializableCheck: false
     });
-    middleware.push(logger);
 
-    // Use Redux DevTools Extension if installed
-    if (window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) {
-      composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
-        // Options: https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md
-        actionCreators: {...actions, push}
+    // Additional development tools
+    if (process.env.NODE_ENV === 'development') {
+      // Logging Middleware
+      const { createLogger } = require('redux-logger');
```

Comment:
does bundling catch conditional requires like this? is there a big cost to just requiring up top?